app/addClothe.tsx

```diff
@@ -1,59 +1,236 @@
-import {Button, StyleSheet, TextInput} from 'react-native';
+import {Button, StyleSheet, TextInput, Image, Text, Platform} from 'react-native';
+import {Picker} from '@react-native-picker/picker';
 import {View} from '@/components/Themed';
 import {useState} from "react";
 import {useNavigation} from "@react-navigation/native";
 import DefaultClothe from "@/classes/clothe";
 import {DatabaseController} from "@/classes/DatabaseController";
+import { Clothe } from '@/classes/clothe';
+import { color } from 'react-native-elements/dist/helpers';
 
 export default function AddClotheScreen() {
     const [clothe] = useState(DefaultClothe());
     const navigation = useNavigation();
+    const [selectedName, setSelectedName] = useState("Clothe Item");
+    const [selectedLink, setSelectedLink] = useState(clothe.link);
+    const [selectedType, setSelectedType] = useState(Clothe.Type.Unknown);
+    const [selectedColor, setSelectedColor] = useState(Clothe.Color.Unknown); // Default value
+    const [selectedSize, setSelectedSize] = useState(Clothe.SleeveSize.Unknown); // Default value
+    
+    const typeKeys = Object.keys(Clothe.Type).filter(key => isNaN(Number(key)));
+    const colorKeys = Object.keys(Clothe.Color).filter(key => isNaN(Number(key)));
+    const sleeveSizeKeys = Object.keys(Clothe.SleeveSize).filter(key => isNaN(Number(key)));
 
-    // Adds clothe to DB and changes screen back to the previous one
+    async function handleCancel() {
```

Comment:
I think this function does not have to be asynchronous.

Reply:
Done